packages/docs/src/routes/tutorial/reactivity/resource/solution/app.tsx

```diff
@@ -27,7 +27,12 @@ export default component$(() => {
       <p>
         <label>
           GitHub username:
-          <input value={github.org} onInput$={(ev, el) => el.value} />
+          <input
+            value={github.org}
+            onInput$={(ev) =>
+              (github.org = (ev.target as HTMLInputElement).value)
```

Comment:
The reason it wasn't working is because github.org wasn't being assigned in onInput$.

However, in your fix you're no longer using the invoking element, which is the correct way to read input events. Please fix this so we can merge.